raincloudy/controller.py

```diff
@@ -163,12 +154,12 @@ def name(self, value):
     @property
     def status(self):
         """Return controller status."""
-        return self.lookup_attr('controller_online')
+        return self.attributes['controller_status']
```

Comment:
The data looked _very_ different than the json file you have in the `test` directory. So I bypassed the `lookup_attr` function entirely.